src/threading/cthread.cpp

```diff
@@ -153,14 +153,13 @@ static size_t global_get_num_threads() {
         str = std::getenv("OMP_NUM_THREADS");
     }
 
-    // If the selected var is unset,
-    //   or no var is set,
-    //   error
+    // If the selected var is unset set the number of threads to
+    // the hint given by the standard library
     if (!str) {
-        std::cerr << "WARNING: Environment variable describing number of threads not set: using 1 thread.\n";
-        std::cerr << "Set one of NMC_NUM_THREADS_VAR, NMC_NUM_THREADS or OMP_NUM_THREADS to set the\n";
-        std::cerr << "number of threads.\n";
-        return 1;
+        auto nthreads = nest::mc::threading::count_available_cores();
```

Comment:
so here's where I'd throw an error if I got a zero (or just throw an error in the other place, and if someone wants to use it with a default of 1, let them catch intentionally and set to 1)